docs/composefs.md

```diff
@@ -21,22 +21,20 @@ At the current time, integration of composefs and ostree is experimental.
 
 ### Enabling composefs (unsigned)
 
-When building a disk image *or* to transition an existing system, run:
+If ostree is compiled with composefs support, then a composefs file
+corresponding to the deployment tree will be generated by default.
+
+The `ostree-prepare-root` binary will look for `ostree/prepare-root.conf` in `/etc` and
+`/usr/lib` in the initramfs. Using that configuration file you can enable composefs.
+This configuration will enable an "unsigned" mode, which does not require fsverity,
+but does make the system more resilient to accidental mutation.
 
 ```
-ostree config --repo=/ostree/repo set ex-integrity.composefs true
+[composefs]
+enabled = yes
 ```
```

Comment:
Feels like it's worth documenting the `maybe` semantics while we're here? Sample docs:

> The value `maybe` is also supported. In that case, `ostree-prepare-root` will enable composefs if a composefs file is found, but otherwise not error out. This is useful in upgrade scenarios where you want to make sure that old systems unable to create composefs files can still upgrade to one that can.

Reply:
We have duplicate docs in `composefs.md` and `man/ostree-prepare-root.xml`. I think the latter is more important. I will try to redirect more from the former.